modules/messages/client/less/thread.less

```diff
@@ -16,11 +16,11 @@
 
   @media (min-width: @screen-xs-min) {
     .content-empty {
-      position: absolute;
-      bottom: 50%;
-      margin-top: -65px;
-      left: 0;
-      right: 0;
+      // position: absolute;
```

Comment:
I think it used to place things in the middle of the screen but has since been broken (should really be done in flex now anyway, but that was back in 2014).